python/tvm/tvmscript/parser.py

```diff
@@ -14,7 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Hybrid Script Parser For TIR"""
+"""TVM Script Parser For TIR"""
```

Comment:
The name tvmscript was a bit duplicated since we are already in the tvm namespace. How about just tvm/script

Reply:
I think being explicit here is helpful. I'd like to avoid the confusing the different scripts available in TVM. We have hybrid script (old) and tvm script. We might have relay script in the future. If I just say script which am I referring to? Also, the agreement in the Discuss thread seems to be tvm script.

Reply:
I agree there might be some confusion that can can discuss further. Since API choice itself is quite important -- we need both conciseness and clarity.

If we want to introduce relay script in the future. Likely we will want to unify with the current tvm script and call them tvm script directly.